content/en/docs/Installation/_index.md

```diff
@@ -340,12 +340,22 @@ kubectl logs -l app=kyverno -n <namespace>
 
 The following flags are used to control the behavior of Kyverno and must be set in the Kyverno ConfigMap.
```

Comment:
"...must be set in the Kyverno ConfigMap."

Are all these flags supported via ConfigMap?

Reply:
I don't think `genWorkers` is supported via ConfigMap, where did you see that?